apps/reference/docs/guides/hosting/overview.mdx

```diff
@@ -111,6 +111,10 @@ For running commands via the Supabase UI.
 
 Supabase Administrative role for maintaining your database.
 
+### Realtime Logs
+
+Running Realtime will generate a lot of redundant database logs. You can mute these logs by setting your database's `log_min_messages` configuration to `fatal`. However, you might miss important log messages, such as database errors, by doing so.
```

Comment:
```suggestion
Set your database's `log_min_messages` configuration to `fatal` to prevent redundant database logs generated by Realtime. However, you might miss important log messages such as database errors. Configure `log_min_messages` based on your needs.
```